testing/upload_validation.ts

```diff
@@ -1152,6 +1153,90 @@ function buildMetadataSchema({sdkVersion}: BuildMetadataSchemaArgs): {
     }),
   ]);
 
+  const contextPropertiesSchema = z.array(propertySchema).min(1);
+
+  const indexedPropertySchema = z.union([
+    propertySchema,
+    z.object({
```

Comment:
Nit, for every object, you probably want to use our wrapper `zodCompleteStrictObject<...>` which will ensure if you add new properties to the type def that TS will catch that you need to update the validator.

Reply:
Done.